spec/features/create_card_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+feature 'Create Card', %q{'It should
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%q` only for strings that contain both single quotes and double quotes.